scripts/llnl/common_build_functions.py

```diff
@@ -123,13 +123,21 @@ def log_failure(prefix, msg, timestamp=""):
     json.dump(info,open(pjoin(prefix,"failed.json"),"w"),indent=2)
 
 
+def assertUberenvExists():
+    if not os.path.exists(get_uberenv_path()):
+        print("[ERROR: {0} does not exist]".format(get_uberenv_path()))
+        sys.exit(1)
```

Comment:
It might be useful to have a message here like we do for BLT when it hasn't been cloned yet,